secure-api-gateway-ob-uk-common-bom/pom.xml

```diff
@@ -39,7 +39,17 @@
         <url>http://www.forgerock.org</url>
     </organization>
 
-    <inceptionYear>2020</inceptionYear>
```

Comment:
Why delete inception year? Isn't it used by the licence checker?

Reply:
It's defined in `secure-api-gateway-parent` is not necessary anymore in the pom of bom module.